designsafe/apps/workspace/models/app_entries.py

```diff
@@ -57,7 +57,7 @@ class AppTrayCategory(models.Model):
         help_text="A category for the app tray.", max_length=64, unique=True
     )
     priority = models.IntegerField(
-        help_text="Category priority, where higher number tabs appear before lower ones.",
+        help_text="Category priority, where lower number tabs appear before higher ones.",
```

Comment:
This will also require a migration.

Reply:
✅ [3eb55e9](https://github.com/DesignSafe-CI/portal/pull/1305/commits/3eb55e917ecbef3171a15da736a16e14a2f3c4ce)